curriculum/challenges/english/07-scientific-computing-with-python/learn-list-comprehension-by-building-a-case-converter-program/657ef2a86d4e545cec9a85fb.md

```diff
@@ -27,6 +29,35 @@ You should write a new `if` statement with `char.isupper()` as the condition. Do
 })
```

Comment:
Please change the previous hint into:

You should write a new `if` statement with `char.isupper()` as the condition. Remember to add the colon at the end and use the `pass` keyword to fill the `if` statement body.

And the test into:

```
({
    test: () => {        
        const commentless_code = __helpers.python.removeComments(code)
        const transformedCode = commentless_code.replace(/\r/g, "");
        const {function_body} = __helpers.python.getDef("\n" + commentless_code, "convert_to_snake_case");
        assert.match(function_body, /\s+if\s+char\.isupper\s*\(\s*\)\s*:\s+pass/);
    }
})
```